core/templates/dev/head/css/oppia.css

```diff
@@ -133,6 +133,11 @@ md-input-group.md-default-theme label {
   margin-right: 20px;
 }
 
+.protractor-test-search-bar-dropdown-menu {
```

Comment:
This class name is used for e2e test, try adding a new class name starts with `oppia-...`

Also, try adding this style to the respective `.html` file.